programs/cw-ics08-wasm-eth/src/contract.rs

```diff
@@ -12,13 +12,16 @@ use ibc_proto::ibc::{
     },
 };
 
-use crate::{custom_query::EthereumCustomQuery, query, state::store_client_state};
+use crate::{custom_query::EthereumCustomQuery, msg::MigrateMsg, query, state::store_client_state};
 use crate::{
     msg::{ExecuteMsg, InstantiateMsg, QueryMsg, SudoMsg},
     state::store_consensus_state,
 };
 use crate::{sudo, ContractError};
 
+const STATE_VERSION: &str = "1.0.0";
+const CONTRACT_NAME: &str = "cw-ics08-wasm-eth";
```

Comment:
Can use crate name as well, should be a macro for it

Reply:
I'll look into that!

Reply:
Fixed